requirements_test.txt

```diff
@@ -1,4 +1,3 @@
--r requirements_test_pre_commit.txt
```

Comment:
I believe the purpose of `requirements_test_pre_commit.txt` is to invalidate the cache in the CI and install + test a new dependency if a user changes the file manually (or dependabot bumps it).
Apart from that, there is a separate job in the CI to run pre-commit and its dependencies are managed via `.pre-commit-config.yml`.